images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -0,0 +1,50 @@
+Import-Module (Join-Path "..\SoftwareReport" "SoftwareReport.Android.psm1") -DisableNameChecking
+
+Describe "Android SDK" {
+    $android = (Get-ToolsetContent).android
+    $androidInstalledPackages = Get-AndroidInstalledPackages
+
+    $platformTestCases = @()
+    $platformList = $android.platform_list
+    $platformList | ForEach-Object {
+        $platformTestCases += @{ platformVersion = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $buildToolsTestCases = @()
+    $buildToolsList = $android.build_tools
+    $buildToolsList | ForEach-Object {
+        $buildToolsTestCases += @{ buildToolsVersion = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $extraPackagesTestCases = @()
+    $extraPackagesList = $android.extra_list
+    $extraPackagesList | ForEach-Object {
+        $extraPackagesTestCases += @{ extraPackage = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $addonsTestCases = @()
+    $addonsList = $android.addon_list
+    $addonsList | ForEach-Object {
+        $addonsTestCases += @{ addonPackage = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    It "Platform version <platformVersion> is installed" -TestCases $platformTestCases {
+        $regex = "platforms;$platformVersion"
+        [boolean]($installedPackages | Where-Object { $_ -match $regex }) | Should -Be $true
```

Comment:
Use built-in parameter: `Should -BeTrue`

Reply:
Done